### ISSUE TITLE
Staging Review finding: "Details" links have the same accessible name

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#136811]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** 1010 Health Apps (33001)
**Product:** Claim Status Tool
**Feature:** Finding Claims

## Findings details

**VA.gov Experience Standard - issue:** [User doesn't have enough information to complete a task.](https://design.va.gov/about/experience-standards/#user-has-enough-information-to-complete-a-task)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)

**Launch-blocking:** No 
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility), @kkitagawa-bosch (IA)

### Description

On "Your claims..." the claim cards "Details" links all have the same accessible name "details", which can be confusing for screen reader users, as the links are repetitive and aren't specific to the action, resulting in a cumbersome experience

Tagging in from IA: This is a problem not only for the accessible names, but also the links in the copy itself. Links need to tell users where they're being taken and what to expect when they select the link 

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="942" height="517" alt="details links" src="https://github.com/user-attachments/assets/90e9cfc2-b012-4eb7-80f1-f49aa271a639" />

</details>

## Recommended action

Ensure that the accessible name of each link is specific to the card, such as "Details, Claim for compensation". This can be done either through explicit link text, an `aria-label` or `aria-labelledby` to provide more context to the links.

Keegan (IA): my preference is towards fixing the issue in the explicit link text. While an aria label helps screen readers, it still leaves contextless links for sighted users 

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** [WCAG 2.4.4 Link Purpose (In Context) - Level A](https://www.w3.org/TR/WCAG22/#link-purpose-in-context)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Context:** N/A
- **VADS Guidance:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoe) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe this finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
